pkg/odo/cli/component/push.go

```diff
@@ -57,9 +57,17 @@ func NewPushOptions() *PushOptions {
 	}
 }
 
+func (po *PushOptions) CompleteDevfilePath() {
```

Comment:
I think this is something that should be more common. But unsure where to put it so keeping it here for now. If anyone has suggessions please provide them :)

Reply:
Why not do it like `url create`? https://github.com/openshift/odo/blob/e08aa97dad5a5c27564e0f08cb46600f8f52d540/pkg/odo/cli/url/create.go#L324
Because `odo delete` doesn't seem to have the `devfile` path flag like create.
```
Flags:
      --context string   Use given context directory as a source for component settings
  -f, --force            Delete url without prompting
  -h, --help             Help for delete
      --now              Push changes to the cluster immediately
```

Reply:
Yes, it does not have the devfile flag, Do you mean to say add devfile flag to delete as well. Ok, i guess we could do that